### PR TITLE
Adding ability to always emit log regardless of category

### PR DESCRIPTION
**Pull request by softwarewright**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ class CategoryLogger extends Logger {
     );
     const minLevel = Logger.levelFromName[levelNameForCategory];
 
-    if (record.level >= minLevel) {
+    if (record.level >= minLevel || record.alwaysEmitLog) {
```
Comment (isaacjwilliams) [inline src/index.js:142]: This feels like `!important` in CSS: an escape hatch for badly factored codebases. 😔 😩

**Review by isaacjwilliams — APPROVED** [review]
YOLO 😜

File: src/index.js (hunk 1 above)
Comment (softwarewright) [reply src/index.js:142]: Yeah I definitely agree with the sentiment, maybe we can chat through other possible solutions.

Comment (aiwilliams) [reply src/index.js:142]: I would like to say that I agree, we need to think through this: https://jupiterone.atlassian.net/browse/DP-1431?focusedCommentId=33796

**Pull request closed without merging**